docs/listeners/custom.rst

```diff
@@ -40,10 +40,9 @@ Creating your own listener class is very similar to using a controller as a list
   <?php
   namespace App\Lib\Listeners;
 
-  use Cake\Event\EventListenerInterface;
   use Cake\Event\Event;
 
-  class MyListener implements EventListenerInterface
+  class MyListener extends BaseListener
```

Comment:
Ah yes, for some reason I saw it was in the same folder, so thought it wouldn't need one. When in fact, it'll not be in the same folder in userland code.

Good spot, thanks!